modules/Core/src/main/java/org/terasology/logic/health/HealthAuthoritySystem.java

```diff
@@ -231,9 +235,14 @@ public void onHeal(DoHealEvent event, EntityRef entity) {
     @ReceiveEvent(components = {HealthComponent.class})
     public void onLand(VerticalCollisionEvent event, EntityRef entity) {
         HealthComponent health = entity.getComponent(HealthComponent.class);
+        CharacterMovementComponent move = entity.getComponent(CharacterMovementComponent.class);
+
+        Optional<Prefab> prefab = Assets.get(new ResourceUrn("engine:player"), Prefab.class);
+        CharacterMovementComponent moveDefault = prefab.get().getComponent(CharacterMovementComponent.class);
+
         float speed = Math.abs(event.getVelocity().y);
 
-        if (speed > health.fallingDamageSpeedThreshold) {
+        if (speed > health.fallingDamageSpeedThreshold*Math.max(move.height/moveDefault.height,1.0f)) {
```

Comment:
I think that this should be moved to a separate line. Also, the damage needs to be adjusted to this variable threshold, not the original value..